Formula/l/lnav.rb

```diff
@@ -1,10 +1,19 @@
 class Lnav < Formula
   desc "Curses-based tool for viewing and analyzing log files"
   homepage "https://lnav.org/"
-  url "https://github.com/tstack/lnav/releases/download/v0.12.0/lnav-0.12.0.tar.gz"
-  sha256 "a6f51a5019df1870c6a5889e337cf2cfc1365b04d81a0602f7f071ec971f5997"
   license "BSD-2-Clause"
 
+  stable do
+    url "https://github.com/tstack/lnav/releases/download/v0.12.1/lnav-0.12.1.tar.gz"
+    sha256 "d4565fbe29ad00e1e00efc6bff68e2068e102ace95ff296610d2bd6c04b13d67"
+
+    # patch to fix static init issue
```

Comment:
Can you link to the issue or PR?
```suggestion
    # patch to fix static init issue
    # https://github.com/???
```

Reply:
@MikeMcQuaid this is a direct commit patch from @tstack, https://github.com/tstack/lnav/commit/884e6b7415c562621487abe922fd3042da71ff32

see discussions above.

Reply:
@chenrui333 That's fine. I just don't know what "static init issue" means and why we want/need this patch, thanks!

Reply:
oh, it caused some crash when starting the app. (I can maybe improve the comment a bit)

Reply:
It was an issue with C++ static initializer ordering.  A variable was being used before it was initialized and caused a crash.